src/discussions/discussions-home/DiscussionSidebar.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import classNames from 'classnames';
+import {
+  Redirect, Route, Switch, useLocation,
+} from 'react-router';
+
+import { Routes } from '../../data/constants';
+import { PostsView } from '../posts';
+import { TopicsView } from '../topics';
+
+export default function DiscussionSidebar({ displaySidebar }) {
```

Comment:
Why not use the hook here?

Reply:
It's dependent on whether we're displaying the content area or not. On mobile when showing content we hide sidebar.

```javascript
  if (displayContentArea) {
    // If the window is larger than a particular size, show the sidebar for navigating between posts/topics.
    // However, for smaller screens or embeds, only show the sidebar if the content area isn't displayed.
    displaySidebar = windowSize.width >= breakpoints.large.minWidth;
  }
  ```